src/benchmark.zig

```diff
@@ -1,3 +1,24 @@
+// This benchmark uses a single client to 1) create `account_count`
```

Comment:
Let's use `//!` to make this into a doc comment.

Reply:
TIL about this. Thanks!